src/protocol/mod.rs

```diff
@@ -20,13 +20,13 @@ use yaml_rust::{yaml::Hash, Yaml, YamlLoader};
 
 #[derive(PartialEq, Eq, Debug, Clone)]
 pub struct Step {
-    pub command: Command,
+    pub command: CommandMatcher,
```

Comment:
i held off on renaming this field because i want to see if we prefer something other than `CommandMatcher`. once we settle on that though, i imagine this field name should update as well.

Reply:
I like `CommandMatcher`.

Reply:
should i change the name of this field from `command` => `command_matcher`? just want to doublecheck. if yes, would you like me to do anything extra WRT backwards compatibility?

Reply:
Yes, please change.

You're asking about backwards compatibility of the library? I don't care about that at all.